tests/accessor/generic_accessor_linearization.h

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
```

Comment:
Probably adding a typical Khronos copyright header here.

Reply:
Added Khronos copyright header here and to other new files